cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -193,6 +194,12 @@ def render_files(reldata=None):
     target = os.path.join(rendered_manifest_dir, 'kubedns-rc.yaml')
     # Render files/kubernetes/kubedns-rc.yaml for the DNS pod.
     render('kubedns-rc.yaml', target, context)
+    # when files change on disk, we need to inform systemd of the changes
+    try:
+        check_call(['systemctl', 'daemon-reload'])
```

Comment:
Truth be told, when I was poking about in the kubernetes-master unit, we appear to be re-rendering the files which is triggering the fsmodified trigger on systemd, and this was it's recommended path to reconcile between the changes and what systemd is expecting.

If there's a better method to do this, i'm happy to patch it